ci/travis_jobs/test_use_cases_data_assimilation.sh

```diff
@@ -19,6 +19,8 @@ echo mkdir -p ${TRAVIS_OUTPUT_BASE}
 mkdir -p ${TRAVIS_OUTPUT_BASE}
 
 ${TRAVIS_BUILD_DIR}/ci/travis_jobs/docker_setup.sh
```

Comment:
You shouldn't run docker_setup.sh in the test use case scripts. The Docker Setup stage ensures that the docker image on DockerHub is up-to-date, so you just need to pull that image. We are trying to avoid rebuilding the docker image for every test.